packages/system/src/outline.ts

```diff
@@ -0,0 +1,40 @@
+import { OutlineKeys } from "./keys";
+import { CSSProperties, ResponsiveStyle } from "./types";
+import { applyResponsiveStyles } from "./responsive";
+
+export type OutlineProps = Partial<
+  CSSProperties<"outline"> &
+    CSSProperties<"outlineOffset"> &
+    CSSProperties<"outlineWidth"> &
```

Comment:
```suggestion
    CSSProperties<"outlineWidth", true> &
```

`outlineWidth` should be able to accept number value